### PR TITLE
Adjusting FE Rules for k8s testbed using FE as daemonset

### PR DESCRIPTION
**Pull request by muhammadimranfarooqi**

### CODE REVIEW

**Comment by muhammadimranfarooqi** [comment]
@amaltaro 

Please review this

**Comment by muhammadimranfarooqi** [comment]
The format is `<service>.<namepsace>.svc.cluster.local` to forward request to k8s cluster

File: frontend/backends-k8s-preprod.txt [hunk 1]
```diff
@@ -1,5 +1,4 @@
-^/auth/complete/t0_reqmon(?:/|$) vocms0132.cern.ch|vocms0731.cern.ch :affinity
-^/auth/complete/reqmon(?:/|$) vocms0132.cern.ch|vocms0731.cern.ch :affinity
+^/auth/complete/t0_reqmon(?:/|$) t0reqmon.tzero.svc.cluster.local
```
Comment (amaltaro) [inline frontend/backends-k8s-preprod.txt:1]: t0reqmon isn't part of any URN/endpoint, so this line can go away.

**Review by amaltaro — APPROVED** [review]
Imran, changes look good to me. I think the `t0_reqmon` line is not needed though.

File: frontend/backends-k8s-preprod.txt (hunk 1 above)
Comment (muhammadimranfarooqi) [reply frontend/backends-k8s-preprod.txt:1]: How about this then? https://cmsweb-testbed.cern.ch/t0_reqmon/data/info

Comment (muhammadimranfarooqi) [reply frontend/backends-k8s-preprod.txt:1]: It does exist

Comment (amaltaro) [reply frontend/backends-k8s-preprod.txt:1]: Yes, you're right. This is what we have in its configuration
```
main.application = "t0_reqmon"
```

It looks good to me then.

**Pull request merged**